src/amigaDebug.ts

```diff
@@ -537,7 +541,6 @@ export class AmigaDebugSession extends LoggingDebugSession {
 		const env = {
 			...process.env,
 			LD_LIBRARY_PATH: ".", // Allow Linux fs-uae to find bundled .so files
-			DYLD_FALLBACK_LIBRARY_PATH: ".", // Allow Mac fs-uae to find bundled .dylib files
```

Comment:
is this no longer needed on mac?

Reply:
No, the CI script now modifies the binary to reference the lib on a relative path.